### ISSUE TITLE
Kompatibilität mit macOS' Bash und Windows' Git Bash herstellen

### ISSUE THREAD

**Issue by katrinleinweber** [TIBHannover/PICA-kulatur#5]
In der Bash bricht es zwischen diesen Zeilen ab:

https://github.com/TIBHannover/PICA-kulatur/blob/c7a7d9b96ba07eba8b23b5b2fb563d8648bedc9a/PICA-kulatur.sh#L58-L59

**Comment by Phu2** [comment]
Kannst Du hier mal so eine Downloaddatei aus der WinIBW einstellen, damit man testen kann?

**Comment by katrinleinweber** [comment]
Ja, habe ich vor. Not sure if schützenswert, daher warte ich noch auf Freigabe meiner Beispieldaten durch die Fachreferent*inn*en.
